webpack.config.prod.js

```diff
@@ -0,0 +1,21 @@
+const merge = require('webpack-merge');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+let prod = {
```

Comment:
const are not exactly constants on JavaScript, I and a lot of people use it only for require o very specific things :)